src/Lecture2.hs

```diff
@@ -175,64 +297,70 @@ challenging. However, you still may find some of them easier than some of the
 previous ones. Difficulty is a relative concept.
 -}
 
-{- | Write a function that takes a list of numbers and returns 'True'
-if all the numbers are in the increasing order (i.e. the list is
-sorted).
-
->>> isIncreasing [3, 1, 2]
-False
->>> isIncreasing [1 .. 10]
-True
--}
+-- | Write a function that takes a list of numbers and returns 'True'
+-- if all the numbers are in the increasing order (i.e. the list is
+-- sorted).
+--
+-- >>> isIncreasing [3, 1, 2]
+-- False
+-- >>> isIncreasing [1 .. 10]
+-- True
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
-
-{- | Implement a function that takes two lists, sorted in the
-increasing order, and merges them into new list, also sorted in the
-increasing order.
-
-The lists are guaranteed to be given sorted, so you don't need to
-verify that.
-
->>> merge [1, 2, 4] [3, 7]
-[1,2,3,4,7]
--}
+isIncreasing [] = True
+isIncreasing [_] = True
+isIncreasing [x, y] = x <= y
```

Comment:
This case is redundant because it's already handled by other cases. It may be a nice optimization but hard to tell without benchmarks 🤔